Packs/XSOARmirroring/Integrations/XSOARmirroring/XSOARmirroring.py

```diff
@@ -332,18 +335,15 @@ def fetch_incidents(client: Client, max_results: int, last_run: Dict[str, int],
 
         incident_result['attachment'] = file_attachments
         incidents_result.append(incident_result)
-        incident_created_time = arg_to_timestamp(
-            arg=incident.get('created'),  # type: ignore
-            arg_name='created',
-            required=True
-        )
+        incident_created_time = dateparser.parse(incident.get('created'))  # type: ignore
 
         # Update last run and add incident if the incident is newer than last fetch
-        if incident_created_time > latest_created_time:
+        if incident_created_time > latest_created_time:  # type: ignore
```

Comment:
Please add the specific error name that you ignoring like here:

```suggestion
        if incident_created_time > latest_created_time:  # type: ignore[operator]
```